run-make-check.sh

```diff
@@ -68,7 +68,13 @@ function run() {
     BUILD_MAKEOPTS=${BUILD_MAKEOPTS:-$DEFAULT_MAKEOPTS}
     CHECK_MAKEOPTS=${CHECK_MAKEOPTS:-$DEFAULT_MAKEOPTS}
 
-    $DRY_RUN ./do_cmake.sh $@ || return 1
+    # Do the right thing if Python 2 is absent - http://tracker.ceph.com/issues/23035
+    CMAKEOPTS="$@"
+    if ! type python2 > /dev/null 2>&1 ; then
+        CMAKEOPTS="-DWITH_PYTHON2=OFF -DMGR_PYTHON_VERSION=3 $CMAKEOPTS"
```

Comment:
Shouldn't we also add `-DWITH_PYTHON3=ON` ?

Reply:
Oops! Good catch.